### PR TITLE
Add web view tab

### PR DESCRIPTION
**Pull request by valentinamackovic**
### Description:

Web view tab was added to the new navigation implementation.

### Release Screenshots / Video:

WebExplore tab: 
Screenshot 2020-02-26 at 14 56 13

### Environment Variables:
 * No change

### CODE REVIEW

File: src/WebExplore/Explore.tsx [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
+    this.props.navigation.setOptions({
+      //navigation.getParam('otherParam', 'A Nested Details Screen'),
       headerLeft: ({}) => {
-        const hasBack = navigation.getParam('hasBack', '0')
-        if (hasBack === '1') {
-          return (
-            <TouchableOpacity onPress={() => {
-              navigation.webView.goBack()
-            }} style={{
+        return (
```
Comment (Krakaw) [inline src/WebExplore/Explore.tsx]: Is there something specific happening here that we don't need the `hasBack` functionality?

Comment (valentinamackovic) [reply src/WebExplore/Explore.tsx]: When I was testing onMessage method wasn't getting executed so hasBack param was never set properly. This way there is always a 'Back' button in navigation header.

Comment (Krakaw) [reply src/WebExplore/Explore.tsx]: You need to uncomment http://github.com/big-neon/bn-web/blob/master/src/components/routes/OnRouteChange.js#L70-L70 and change it to 
`sendReactNativeMessage(this.props.location);`

Comment (Krakaw) [reply src/WebExplore/Explore.tsx]: Though if the functionality still works and we don't get weird navigation issues with the back button there, I'm happy to leave it in.

Comment (valentinamackovic) [reply src/WebExplore/Explore.tsx]: As far as i noticed everything works good. When the base screen is being shown click on the 'Back' button does nothing. I didn't encounter any issues.

**Pull request merged**